Documentation/guides/DotNet5.md

```diff
@@ -45,6 +45,13 @@ In .NET 5 the behavior of the following MSBuild tasks will change, but
 better align with existing .NET 5 semantics, at the cost of compatibility with
 previous Xamarin.Android releases.
 
+`$(AndroidClassParser)` will be `class-parse` by default.
+
+`$(AndroidCodegenTarget)` will be `XAJavaInterop1` by default.
+
+If Java binding is enabled with `@(InputJar)` or `@(EmbeddedJar)`,
+then `$(AllowUnsafeBlocks)` must default to `True`.
```

Comment:
It might be handy to list `@(LibraryProjectZip)` here too since binding `.aar` libraries is probably fairly common and I think this PR already supports it. (`<CreateLibraryResourceArchive/>` will copy the `.jar` files from within the `.aar` archives to `$(IntermediateOutputPath)library_project_jars`, and `AddLibraryJarsToBind` will add them to `@(InputJar)`.)

For the sake of completeness, `@(LibraryProjectProperties)` could also be added. I doubt any users will be trying to bind those old `android create lib-project` library projects anymore, but it might be handy to have it listed for the teams own future reference.

Reply:
It looks like `AddLibraryJarsToBind` runs first and so `@(InputJar)` is filled out. Otherwise, we wouldn't get any C# bindings for them.

I don't think we need to list these?

Reply:
The main thing I was thinking was that a user reading this document might think `@(LibraryProjectZip)` was not supported outside of bindings projects and that they'd need to keep creating bindings projects to use `@(LibraryProjectZip)`.

Maybe that info actually makes more sense in its own section anyway? So maybe this PR could add another short section with its own heading like "Consolidation of bindings projects and other project types" or "Changes to bindings projects" that says something like:

> Bindings projects are no longer a separate project type. All of the bindings project build actions like **LibraryProjectZip** and **TransformFile** can now be used directly in Xamarin.Android class library projects and Xamarin.Android application projects instead.

Reply:
Sorry, I realize now your commenting on the doc and not the code. 🤦‍♂️ That makes sense, thanks.

Reply:
Ah, ha! No worries. The new additions to the doc from the follow-up commits look good!